post-processing/streamlit_post_processing.py

```diff
@@ -177,6 +177,16 @@ def axis_select(label: str, axis: dict):
 
     # units select
     units_select(label, axis)
+    
+    # axis range
+    use_default_ranges = st.checkbox("Use default ranges", axis.get("range").get("use_default"), key="{0}_axis_range_use_default".format(label))
+    if not use_default_ranges:
+        axis_range_min, axis_range_max = st.columns(2)
+        with axis_range_min:
+            st.number_input("Minimum", key="{0}_axis_range_min".format(label))
+        with axis_range_max:
+            st.number_input("Maximum", key="{0}_axis_range_max".format(label))
```

Comment:
From a UX standpoint it's a little unclear to have these min/max options available for generic graphs when they do precisely nothing for any plot that isn't a line graph at the moment. It's not a major problem and doesn't need to be addressed in this PR, but worth either opening a new issue or adding to an existing "fix UI" issue when this PR gets merged.

Reply:
Good point, I believe you can dynamically hide them depending if plot_type = line is selected. Would that work?

Reply:
Yes, that's what I was thinking.